content/discrete-math/concepts/enumeration/enumeration.md

```diff
@@ -0,0 +1,63 @@
+---
+Title: 'Enumeration'
+Description: 'Enumeration is a complete, ordered listing of all the items in a collection.'
+Subjects:
+  - 'Computer Science'
+  - 'Discrete Math'
+Tags:
+  - 'Arithmetic'
+  - 'Math'
+  - 'Logical'
+  - 'Logic'
+CatalogContent:
+  - 'computer-science'
+  - 'paths/computer-science'
+---
+
+**Counting Theory** is a part of 'discrete mathematics' that deals with counting the number of possible outcomes of a given event. Being an essential element of 'probability', it tries to answer an abstract question, "How many?"
+
+**Enumeration** is a complete, ordered listing of all the items in a collection. When speaking in the context of 'mathematics' and 'computer science', listing all the elements of a set comes into play.
```

Comment:
```suggestion
**Enumeration** is another word for count. In the context of mathematics, enumerating is a type of counting given a specified event or situation. An enumeration is a complete, ordered listing of all the items in a collection. When speaking in the context of mathematics and computer science, listing all the elements of a set comes into play.

**Counting Theory** is a part of discrete mathematics that deals with counting the number of possible outcomes of a given event. Being an essential element of probability, it tries to answer an abstract question, "How many?"
```

I think your definition for counting theory is spot on! Enumerating is synonymous with counting and so I think in the context of discrete math (the topic), that should be mentioned. Then, it would be good to go into counting theory.